packages/backend-core/src/constants/misc.ts

```diff
@@ -11,24 +11,7 @@ export enum Cookie {
   OIDC_CONFIG = "budibase:oidc:config",
 }
 
-export enum Header {
```

Comment:
This has been exported to `shared-core` to be reused in `frontend-core`, removing magic strings